### ISSUE TITLE
Build an Atom - Adapted "Teacher Tips" style Document Needed for Grant Proposal - by August 11th

### ISSUE THREAD

**Issue by emily-phet** [phetsims/tasks#880]
I need a three-page description of a design concept for an accessible version of Build an Atom, in the style of a teacher tips document (screenshots, with bubbles pointing/indicating a specific feature, with some additional descriptive text). 

I will also need a slightly edited version of Build an Atom's existing teacher tips document.

This will need to be a collaboration between someone on the team familiar with teacher tips documents, and @terracoda - who can help with the descriptive text. 

I'm putting together an [outline for this](https://docs.google.com/a/colorado.edu/document/d/1fShECkaz09Nrn7l5LgAdkem3nOtTUokujllfonKrNdg/edit?usp=sharing), with more info on specifics.

**Comment by terracoda** [comment]
@emily-phet, I found a copy of the current Teacher Tips for Build an Atom, but I have not found an original design file that can be edited. Your outline is great, but I will likely need help with a some of the visuals that do not yet exist.

**Comment by ariel-phet** [comment]
@arouinfar do you mind helping with this? Obviously it is a short timeline, so high priority

**Comment by emily-phet** [comment]
@terracoda Just to be clear, my thought is for you to help with the wording of the descriptions (e.g., what's in the bubbles of text). Others on the team have experience creating teacher tips, which is what this is very similar to, but no one has more experience with potential accessibility features than you. I may not be able to give feedback as quickly as I would like, so it would be helpful for you to step in and keep things moving.

**Comment by terracoda** [comment]
@emily-phet, understood. I can focus on the bubbles, and keep things moving.

**Comment by arouinfar** [comment]
@emily-phet I'll get started on the tips documents. Thanks for providing an outline!

**Comment by arouinfar** [comment]
@emily-phet I have a question regarding:

> Move “See all activities for Build an Atom” up closer to list above. Remove “here” and replace with text link to sim page - https://phet.colorado.edu/en/simulation/build-an-atom. 
Add sentence below “See all activities for Build an Atom” that reads “For an introduction to Build an Atom, and more tips for use, watch the Video Primer at https://phet.colorado.edu/en/simulation/build-an-atom.”

I've noticed that these links are directed to the sim page, but the "here" link actually goes to the For Teachers section of the sim page that includes the activities and primers.
https://phet.colorado.edu/en/simulation/build-an-atom#for-teachers-header

Which would you prefer?

**Comment by arouinfar** [comment]
@emily-phet here's the modified Teacher Tips. 
[build_an_atom_html5_guide-GrantProposal.pdf](https://github.com/phetsims/tasks/files/1205498/build_an_atom_html5_guide-GrantProposal.pdf)

**Comment by arouinfar** [comment]
@emily-phet @terracoda here's a rough draft of the document highlighting the a11y features. I still need to work on a mockup for the sim options dialog, but I've included everything else that @emily-phet outlined in the doc. I threw in some screenshots so there'd be context for the callouts, but they can be easily replaced with something else. 

I've included an editable version in case you want to do edit any of the text directly.
[build_an_atom_a11y-GrantProposal.pptx](https://github.com/phetsims/tasks/files/1205734/build_an_atom_a11y-GrantProposal.pptx)
[build_an_atom_a11y-GrantProposal.pdf](https://github.com/phetsims/tasks/files/1205738/build_an_atom_a11y-GrantProposal.pdf)

**Comment by terracoda** [comment]
@arouinfar, I am reading through mock-ups :-)

**Comment by emily-phet** [comment]
@arouinfar Thanks! The modified Build an Atom teacher tips look great.
For the accessible sim version
All pages: 

- Please add "Accessible Simulation Concept: Build an Atom" to the header (in that black header bar). 
- Add the little keyboard help button to the bottom bar of each screenshot.

Alternative Input Section: 
- Please add another callout 'pointer' lines to the "all interactive objects accessible through alternative input". Like second pointy line to the Element box. Also, change wording to "Access all interactive objects through alternative input" with "Access" in bold.

Auditory Display Section:

- Add "(non-speech sound)" after "sonification" in the description
- It would be great to have some small overlays, that are different from the purple bubbles that indicat speech or sound - like having more of a white background, comic book looking bubble that has italicized text that represents what would be read out. Something for verbal descriptions, and for sound. @terracoda thoughts on how this might work?

**Comment by terracoda** [comment]
@emily-phet:

- For the alternative input section, the action verb, "Access" could also be "Operate". And I would recommend a second pointy line to either the checkboxes or the radio buttons, or even to the Atom and or the buckets.
- For keyboard call-out, I suggest: "Review common and custom keyboard shortcuts." Or "Review common and custom key commands with the keyboard shortcuts button."

**Comment by terracoda** [comment]
@emily-phet and @arouinfar, mayb ein some screenshots, we do not have to have all accordions open. That might indicate how they are interactive, and help focus the accompanying description (if needed).

**Comment by terracoda** [comment]
@arouinfar and @emily-phet for the Auditory display section, I think it might make sense to flip the order of the call-outs around, and keep the text stuff on the left and sonfication stuff on the right.

For example, Text description on left side could look like this:

1. Hear a "big-picture" overview that describes the essential state of the sim, or explore details of each object any time, all accessible with a screen reader.
2. On-screen headings and labels are read aloud through text-to-speech outlining content and interactions. 
3. Hear action-oriented descriptions for interactive objects as you explore the sim.

**Comment by terracoda** [comment]
@emily-phet, and @emily-phet not sure all that can fit.

**Comment by terracoda** [comment]
@arouinfar, I am trying my suggestions out in the power-point. Will add link when done.

**Comment by arouinfar** [comment]
Thanks @terracoda! I'll review and implement your and @emily-phet's suggestions later this afternoon.

**Comment by terracoda** [comment]
@arouinfar, I also like @emily-phet suggestion of making the call-outs different for text-to-speech. That idea might work better.

**Comment by terracoda** [comment]
@arouinfar, not sure this is any better ;-)
[build_an_atom_a11y-GrantProposal-TS-edits-v2.pptx](https://github.com/phetsims/tasks/files/1206177/build_an_atom_a11y-GrantProposal-TS-edits-v2.pptx)

**Comment by emily-phet** [comment]
@terracoda @arouinfar Thanks! I combined the last two version to come up with this (attached). It needs the keyboard shortcuts button in the mockups, and a sound on/off button. @arouinfar, can you add these, and mockup something for the visual display concept screens, and the control panel? Probably the control panel screen will take the most work, so if you could mockup something for that (you could start with the keyboard shortcuts modal). If you could do that in an easily editable format (like the powerpoint files you shared) that would be helpful in case we don't finish this today.

[build_an_atom_a11y-GrantProposal-EM.pptx](https://github.com/phetsims/tasks/files/1211741/build_an_atom_a11y-GrantProposal-EM.pptx)

**Comment by terracoda** [comment]
@emily-phet, nice job! The little sound icons around "Lithium" is a nice touch.

In the visual zoom section, do you want the pink highlight around the letters, or around the heading. The pink normally goes around the focused element which in this case, i think is the interactive heading that opened the accordion.

![baa-net-charge-highlight](https://user-images.githubusercontent.com/1902116/29127774-a0d75b10-7cfc-11e7-8e82-89aebeca810a.png)

**Comment by terracoda** [comment]
@emily-phet, oh, I see, you are highlighting the auditory display, not keyboard focus.

**Comment by arouinfar** [comment]
Thanks @emily-phet and @terracoda! I'll be focusing on this issue once my morning meetings wrap up.

**Comment by arouinfar** [comment]
Converting outstanding work to a checklist...
- [ ] Add the little keyboard help button to the bottom bar of each screenshot.
- [ ] Alternative Input Section: Please add another callout 'pointer' lines to the "all interactive objects accessible through alternative input". Like second pointy line to the Element box. 
- [ ] For the alternative input section, the action verb, "Access" could also be "Operate". And I would recommend a second pointy line to either the checkboxes or the radio buttons, or even to the Atom and or the buckets.
- [ ] mock the control panel
- [ ] mockup something for the visual display concept screens

@emily-phet what are you looking for for the last item in this list?  I'm working through the other things now.

**Comment by emily-phet** [comment]
Thanks @arouinfar

**Comment by emily-phet** [comment]
Whoops! Did that accidentally from my phone!

Anyway, thanks! In the google doc with pictures of my drawings, I think the second to last one shows the visual display example. Essentially, the entire screen is one of the accordion boxes (so imagine a student has selected to zoom in on only one of the accordion boxes. Let me know if that helps, there are some example call outs in my drawing also.

**Comment by arouinfar** [comment]
@emily-phet I made a large version of the Net Charge panel. Let me know if I can tweak it.
![image](https://user-images.githubusercontent.com/8419308/29142011-b8114438-7d0d-11e7-9ee1-0c866e58f939.png)

I didn't mock anything up for the high contrast visual display, because I'm not sure what that should look like.

**Comment by arouinfar** [comment]
I've mocked up a control panel in PowerPoint, but it looks a bit rougher than it would in Illustrator... 

[build_an_atom_a11y-GrantProposal-AR.pptx](https://github.com/phetsims/tasks/files/1212782/build_an_atom_a11y-GrantProposal-AR.pptx)

**Comment by emily-phet** [comment]
Thanks, this is great! I know you're heading out soon, so I think we can take it from here @arouinfar.

**Comment by arouinfar** [comment]
Sounds good, thanks @emily-phet!

**Comment by emily-phet** [comment]
Just a note, in case it ever comes up again...
For the color contrast screenshot of the sim, I just take a screenshot, open in Preview, and adjust the color under Tools>Adjust Color...
Unfortunately, if you take a screenshot while in the accessibility invert colors mode, the color change is not recorded so you have to take an image and manually adjust the colors in the file.

**Comment by terracoda** [comment]
@emily-phet, I think you need a sound button, too, like the one in John Travoltage or Ohm's Law.

**Comment by emily-phet** [comment]
Thanks! All done.
